java/compaction/compaction-core/src/main/java/sleeper/compaction/strategy/LeafPartitionCompactionStrategy.java

```diff
@@ -28,5 +27,5 @@ public interface LeafPartitionCompactionStrategy {
 
     void init(InstanceProperties instanceProperties, TableProperties tableProperties, CompactionJobFactory factory);
 
-    List<CompactionJob> createJobsForLeafPartition(Partition partition, List<FileReference> activeFilesWithNoJobId);
+    List<CompactionJob> createJobsForLeafPartition(String partition, List<FileReference> filesInAscendingOrder);
```

Comment:
I can't see from the name of the second parameter which files I should expect to be there.